app/uk/gov/hmrc/apiscope/controllers/package.scala

```diff
@@ -55,16 +55,16 @@ package object controllers extends ApplicationLogger {
     )
   }
 
-  def validationResult(errors: Seq[(JsPath, Seq[JsonValidationError])]): JsValue = {
+  def validationResult(errors: collection.Seq[(JsPath, collection.Seq[JsonValidationError])]): JsValue = {
```

Comment:
We've been trying to avoid this by importing the immutable seq or even switching to a list.

Reply:
This is for a reason, if I change this to immutable seq or a list, then I need to make a lot changes in the other parts of the code just to accommodate this change; which I tried to avoid, also this method is being used in only one place.

I have used immutable seq in other methods instead, tried keep changes as minimal as possible.